bot/information/information.py

```diff
@@ -1,47 +1,82 @@
+import pybamm
+
+
 def information(
     chemistry,
     model,
-    solver,
     is_experiment,
     cycle,
     number,
-    is_comparison
+    is_comparison,
+    param_to_vary
 ):
     """
     Generates tweet text.
     Parameters:
         chemistry: dict
-        model: pybamm.BaseModel
-        solver: pybamm.BaseSolver
+        model: pybamm.BaseModel or dict
         is_experiment: bool
-        cycle: list
-        number: numerical
+        cycle: list or None
+        number: numerical or None
         is_comparison: bool
+        param_to_vary: str or None
     Returns
         tweet_text: str
     """
 
-    if is_experiment:
-        tweet_text = (
-            str(cycle)
-            + " * "
-            + str(number)
-            + " "
-            + str(model.name)
-            + " "
-            + str(chemistry["citation"])
-        )
-        return tweet_text
+    params = pybamm.ParameterValues(chemistry=chemistry)
+    c_rate = (
+        params["Nominal cell capacity [A.h]"] / params["Current function [A]"]
```

Comment:
should this fraction be the other way? higher current should mean a higher c-rate

Reply:
Yup, you are right. Sorry Saransh, got it the wrong way around

Reply:
No worries, I'll update this